app/lib/util/__tests__/error-tracking-spec.js

```diff
@@ -225,6 +225,54 @@ describe('error-tracking', function() {
     // then
     expect(sentryCloseSpy).to.have.been.called;
   });
+
+
+  describe('#setTag', function() {
+
+    it('should set tag when error tracking is active on start', function() {
+
+      // given
+      const setTagSpy = sinon.spy();
+
+      const config = mockConfig({ 'editor.privacyPreferences': { 'ENABLE_CRASH_REPORTS': true } });
+      const flags = mockFlags();
+      const renderer = mockRenderer();
+      const Sentry = mockSentry({ setTagSpy });
+
+      // when
+      errorTracking.start(Sentry, 'v2', config, flags, renderer);
+      errorTracking.setTag(Sentry, 'key', 'value');
+
+      // then
+      expect(setTagSpy).to.have.been.calledWith({ key: 'key', value: 'value' });
+    });
+
+
+    it('should set tag when error tracking is restarted', function() {
+
+      // given
+      const rendererSpy = sinon.spy();
+      const setTagSpy = sinon.spy();
+
+      const config = mockConfig({ 'editor.privacyPreferences': { 'ENABLE_CRASH_REPORTS': false } });
+      const flags = mockFlags();
+      const renderer = mockRenderer(rendererSpy);
+      const Sentry = mockSentry({ setTagSpy });
+
+      // when
+      errorTracking.start(Sentry, 'v2', config, flags, renderer);
+
+
+      const calls = rendererSpy.getCalls();
+
+      errorTracking.setTag(Sentry, 'key', 'value');
+
+      calls[0].args[1]();
+
+      // then
+      expect(setTagSpy).to.have.been.calledWith({ key: 'key', value: 'value' });
```

Comment:
Is it maybe better if we go with `calledOnceWith` here? What we really need to see is that setTag method of Sentry is called **only** when `errorTracking.isActive` is true.

Reply:
In other words: We want to make sure `Sentry.configureScope` is not called when `errorTracking.setTag` is called and `errorTracking.isActive` is false.

Reply:
I've added a new assertion for that:

https://github.com/camunda/camunda-modeler/pull/1823/commits/4de1e85ee6fc832fcb3d8ffb9a17cd47ff5b9b50#diff-300467e0a9b6480b770adcf53018d2fbR269-R270

Reply:
👍

Reply:
The tests seem to fail. Could you check?

Reply:
yeah 🗡️

Reply:
I think we need to reset errorTracking status within a beforeEach function.

Reply:
By errorTracking status I mean: isActive variable.

Reply:
I suggest either converting errorTracking to a class, or exposing another API (resetStatus) which sets isActive to false and makes additionalTags empty. We'd use this before each test.

Reply:
Fixed via https://github.com/camunda/camunda-modeler/pull/1823/commits/29ab6c57827d463d625f99dc9df27ea5009db4f3